index.js

```diff
@@ -5,19 +5,37 @@ const postcss = require('postcss')
 const assert = require('assert')
 const crypto = require('crypto')
 const xtend = require('xtend')
+const stackTrace = require('stack-trace')
+const cssResolve = require('style-resolve').sync
+const fs = require('fs')
+const path = require('path')
 
 module.exports = sheetify
 
 // transform css
 // (str, str, obj?, fn) -> str
 function sheetify (src, filename, options, done) {
   // handle tagged template calls directly from Node
-  if (Array.isArray(src)) src = src.join('')
+  const isTemplate = Array.isArray(src)
+  if (isTemplate) src = src.join('')
   assert.equal(typeof src, 'string', 'src must be a string')
   src = src.trim()
 
+  // Ensure prefix is always correct when run from inside node
+  let css
+  if (!isTemplate && !filename) {
+     // module or file name via tagged template call
+    const callerDirname = path.dirname(stackTrace.get()[1].getFileName())
```

Comment:
ARG! you are right, this won't quite work.  If you pass options is screws things up.  Working on a fix.